docs/source/tutorials/describing_tasks.ipynb

```diff
@@ -1,20 +1,563 @@
 {
```

Comment:
<p>I did not find the FOO': 'foo', 'BAR': 'bar' very illustrative. I was left with a <code>foo - bar - /bin/bash</code> line that told me little about setting variables, expecially becuase the rest of the variables were in the <code>CAPITAL NAME: value</code> format. Should we expand on the example and so more meaningful variables?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<blockquote>expecially becuase the rest of the variables were in the <code>CAPITAL NAME: value</code> format </blockquote><p>I don't understand that part of the comment, as <code>FOO:foo</code> and <code>BAR:bar</code> obviously follow that very<code>CAPITAL NAME: value</code> format?  But sure, one can use different variable names.  <code>FOO</code> and <code>BAR</code> are very common names for simple examples though, just like <code>alice and bo</code><a href="https://en.wikipedia.org/wiki/Foobar" target="_blank">here</a> see [here](https://en.wikipedia.org/wiki/Foobar).</p>

Reply:
Here is the output:
```
foo - bar - /bin/bash
RP_APP_TUNNEL_ADDR=144.76.72.175:27017
RP_BOOTSTRAP_0_REDIR=True
RP_GTOD=/home/merzky/radical.pilot.sandbox/rp.session.rivendell.merzky.019441.0011//pilot.0000//gtod
RP_PILOT_ID=pilot.0000
RP_PILOT_SANDBOX=/home/merzky/radical.pilot.sandbox/rp.session.rivendell.merzky.019441.0011//pilot.0000/
RP_PROF=/home/merzky/radical.pilot.sandbox/rp.session.rivendell.merzky.019441.0011//pilot.0000//prof
RP_PROF_TGT=/home/merzky/radical.pilot.sandbox/rp.session.rivendell.merzky.019441.0011//pilot.0000//task.000065/task.000065.prof
RP_RANK=0
RP_RANKS=1
RP_RESOURCE=local.localhost
RP_RESOURCE_SANDBOX=/home/merzky/radical.pilot.sandbox
RP_SESSION_ID=rp.session.rivendell.merzky.019441.0011
RP_SESSION_SANDBOX=/home/merzky/radical.pilot.sandbox/rp.session.rivendell.merzky.019441.0011/
RP_TASK_ID=task.000065
RP_TASK_NAME=task.000065
RP_TASK_SANDBOX=/home/merzky/radical.pilot.sandbox/rp.session.rivendell.merzky.019441.0011//pilot.0000//task.000065
```

The first line makes little sense compared to the others, and even less sense to the user IMHO. Does that help?

Reply:
Yes, that helps.  We can change the command to `echo "TEST OUTPUT: $FOO - $BAR - $SHELL"` so that it's clearer where that first line comes from.  Would that help IYO?

Reply:
It would be a good start but maybe we can find some 'interesting'/'meaningful' variables to add instead of the two generic FOO/BAR? That would require to imagine/leverage our experience about what a potential user would find Interesting and meaningful.

Reply:
I added some real-world examples.